src/rtc/h5_prep.py

```diff
@@ -576,3 +816,124 @@ def save_hdf5_dataset(ds_filename, h5py_obj, root_path,
 
     del gdal_ds
 
+
+def get_rfi_metadata_dict(burst_in,
+                          rfi_root_path = 'RTC/metadata/QA/RFIInformation'):
+    '''
+    Populate the RFI information into HDF5 object
+
+    Parameters
+    ----------
+    burst_in: Sentinel1BurstSlc
+    rfi_root_path: str
+
+    '''
+    rfi_metadata_dict = {}
+
+    is_rfi_info_empty = burst_in.burst_rfi_info is None
+    rfi_metadata_dict[f'{rfi_root_path}/isRFIInfoAvailable'] =\
```

Comment:
```suggestion
    rfi_metadata_dict[f'{rfi_root_path}/isRfiInfoAvailable'] =\
```
This follows the convention you are using below.